x-pack/plugins/maps/public/connected_components/map/features_tooltip/feature_geometry_filter_form.js

```diff
@@ -80,10 +77,8 @@ export class FeatureGeometryFilterForm extends Component {
 
     // Ensure filter will not overflow URL. Filters that contain geometry can be extremely large.
     // No elasticsearch support for pre-indexed shapes and geo_point spatial queries.
-    if (
-      window.location.href.length + rison.encode(filter).length + META_OVERHEAD >
-      urlOverflow.failLength()
-    ) {
+    // TODO: should we expose the max from core?
+    if (window.location.href.length + rison.encode(filter).length + META_OVERHEAD > 25000) {
```

Comment:
Where did 25000 come from? Maybe this can be imported?

Reply:
Ah yes forgot to follow up on that TODO.